test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -122,6 +128,18 @@ func describeQatDpdkPlugin() {
 
 			framework.Logf("cpa_sample_code output:\n %s", output)
 		})
+
+		ginkgo.It("deploys a crypto pod (dpdk crypto-perf) requesting QAT resources", func() {
+			ginkgo.By("submitting a crypto pod requesting QAT resources")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "apply", "-k", filepath.Dir(cryptoTestGen4YamlPath))
+
+			ginkgo.By("waiting the crypto pod to finish successfully")
+			e2epod.NewPodClient(f).WaitForSuccess("qat-dpdk-test-crypto-perf-tc1-gen4", 300*time.Second)
+
+			output, _ := e2epod.GetPodLogs(f.ClientSet, f.Namespace.Name, "qat-dpdk-test-crypto-perf-tc1-gen4", "openssl-qat-engine")
+
+			framework.Logf("cpa_sample_code output:\n %s", output)
```

Comment:
the references to `openssl-qat-engine` and `cpa_sample_code` look odd

Reply:
Thanks! Fixed!